benchmarks/sparse/dlmc/spmv.py

```diff
@@ -0,0 +1,113 @@
+# Sparse benchmarks
+
+# This benchmark is for Sparse matrix-vector multiplication (SPMV) performance test. 
+# They exist for comparing the performance of sparse matrix routines
+# torch.matmul(sparse, vector)` with different backends (CPU/CUDA)
+# and with other frameworks such as scipy. 
+
+import sys
+from pathlib import Path
+import pandas as pd
+import argparse
+import torch.utils.benchmark as benchmark_utils
+from .utils import load_spmv_dataset, to_coo_scipy, scipy_coo_matmul
+import pickle
+
+
+if __name__ == '__main__':
+
+    path = Path()
+    parser = argparse.ArgumentParser(description='spmv Bench')
+
+    parser.add_argument('--path', type=str, help='dataset path')
+    parser.add_argument('--dataset',
+                        type=str,
+                        help='dataset name (random_pruning, variational_dropout, magnitude_pruning, \
+                            extended_magnitude_pruning)',
+                        default='random_pruning')
+    parser.add_argument('--operation',
+                        type=str,
+                        help='matmul',
+                        default='matmul')
```

Comment:
you are not providing functions to benchmark backward, is it because backward is not implemented for spmv?

Reply:
Yes, it's seems there is an issue for backward for `torch.matmul` when the input is `spase @ vector`.  Moreover there is no test for this case in the `test_autograd.py`